server/cmd/mmctl/commands/version.go

```diff
@@ -51,14 +50,50 @@ type Info struct {
 	Platform     string
 }
 
-func getVersionInfo() Info {
-	return Info{
+func getVersionInfo() (*Info, error) {
+	info, ok := debug.ReadBuildInfo()
+	if !ok {
+		return nil, errors.New("failed to get build info")
+	}
+
+	var (
+		revision     = "dev"
+		gitTreeState = "dev"
+		buildTime    = "dev"
+
+		os       string
+		arch     string
+		compiler string
+	)
+
+	for _, s := range info.Settings {
+		switch s.Key {
+		case "vcs.revision":
+			revision = s.Value
```

Comment:
Does this get the latest tag? If you install `master`, would it register version as the latest tag? (in your example it looks like `v7.11` hence I'm asking)

Reply:
`revision` is always the HEAD commit sha. Maybe you are referring to `Version`, which is the latest version listed in `model/versio.go`? That is curtently v7.11.0 (and need to get updated seperatetly).

Reply:
Right, thanks for the clarification.